repose-aggregator/components/filters/regex-rbac-filter/src/main/resources/META-INF/schema/config/regex-rbac.xsd

```diff
@@ -0,0 +1,155 @@
+                        Default is 0.3
+                    </html:p>
+                </xs:documentation>
+            </xs:annotation>
+        </xs:attribute>
+        <xs:attribute name="component-name" type="xs:string" use="optional" default="regex-rbac">
+            <xs:annotation>
+                <xs:documentation>
+                    <html:p>
+                        Sets the delegation component name for this validator.
+                    </html:p>
+                </xs:documentation>
+            </xs:annotation>
+        </xs:attribute>
+    </xs:complexType>
+
+    <xs:simpleType name="QualityType">
+        <xs:restriction base="xs:double">
+            <xs:minInclusive value="0"/>
+            <xs:maxInclusive value="1.0"/>
+        </xs:restriction>
+    </xs:simpleType>
+
+    <xs:complexType name="ResourcesType">
+        <xs:annotation>
+            <xs:documentation>
+                <html:p>
+                    This is a list of resources, HTTP methods, and roles allowed to access them.
+
+                    These are expressed in a Domain Specific Language (DSL) that is column based with each column
+                    separated by one or more whitespace characters.
+
+                    The first column contains a
+                    <html:a href="http://docs.oracle.com/javase/8/docs/api/java/util/regex/Pattern.html">
+                        Java Regular Expression
+                    </html:a>
+                    defining the resource.
+
+                    The second column contains the HTTP methods that pertain to the resource defined in the first column.
+                    The special HTTP methods ANY and ALL are synonymous and indicate the HTTP method should not play a
+                    part in the RBAC determination.
+
+                    The third and last column contains the roles that must be present in the <code>X-Roles</code> request
+                    header. To support roles with spaces, the non-breaking space (NBSP, &#xA0;) character is used.
+
+                    The RegEx RBAC resources can be either linked via an href attribute or defined directly in this
+                    element.
+                </html:p>
+            </xs:documentation>
+        </xs:annotation>
+
+        <xs:simpleContent>
+            <xs:extension base="ResourcesString">
+                <xs:attribute name="href" type="xs:anyURI" use="optional">
+                    <xs:annotation>
+                        <xs:documentation>
+                            <html:p>
+                                Specifies a location to an external file which contains the RegEx RBAC resources.
+                                If the resources element has a value and the href attribute is configured,
+                                then the RegEx RBAC filter will use the contents of the referenced location for the
+                                endpoints configuration.
```

Comment:
It appears that the filter will actually prefer the inline resources to any externally referenced resources (which totally seems like the right call to me, but is not how I interpret this blurb).

Reply:
There is no preference, they are simply mutually exclusive ways of representing the resources.

Reply:
Derp. I didn't look for the assertion, just looked at the code. Fair enough. In that case, the whole "If the resources element has a value and the href attribute is configured..." bit here is probably unnecessary.

Reply:
Updated to make it more clear.